dev/promote_bom.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+#
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import yaml
+
+from generate_bom import BomGenerator
+from spinnaker.run import run_quick
+
+VERSION = 'version'
+
+class BomPromoter(BomGenerator):
+
+  def __init__(self, options):
+    self.__rc_version = options.rc_version
+    self.__bom_dict = {}
+    super(BomPromoter, self).__init__(options)
+
+  def __unpack_bom(self):
+    """Load the release candidate BOM into memory.
+    """
+    # TODO(jacobkiefer): Replace this command with hal.
+    bom_yaml_string = run_quick('cat 0.3.0-184.yml', echo=False).stdout.strip()
```

Comment:
You can replace this now if you like, `hal versions bom 0.3.0-184 --color false`